apps/admin-x-activitypub/src/components/Search.tsx

```diff
@@ -141,7 +141,7 @@ const Search: React.FC<SearchProps> = ({}) => {
                 {showLoading && (
                     <LoadingIndicator size='lg'/>
                 )}
-                {showNoResults && (
+                {showNoResults && (query.length > 0) && (
```

Comment:
We could move this extra conditional into the `showNoResults` variable to keep it concise

How come you needed to add `query.length > 0` anyways?